cmd/deploy/endpoints.go

```diff
@@ -227,7 +227,7 @@ func (dc *EndpointGetter) showEndpoints(ctx context.Context, opts *EndpointsOpti
 		}
 	default:
 		if len(eps) == 0 {
-			oktetoLog.Printf("There are no available endpoints for '%s'\n", opts.Name)
+			fmt.Errorf(oktetoErrors.ErrEndpointsNotAvailable, opts.Name)
 		} else {
 			oktetoLog.Information("Endpoints available:")
```

Comment:
@nitishfy maybe following this pattern using `oktetoLog.Information` you can print the new message instead of using `fmt.Error`

Reply:
Updated!